databricks/koalas/internal.py

```diff
@@ -60,9 +61,7 @@
 
 NATURAL_ORDER_COLUMN_NAME = "__natural_order__"
 
-HIDDEN_COLUMNS = set([NATURAL_ORDER_COLUMN_NAME])
-
-IndexMap = Tuple[str, Optional[Tuple[str, ...]]]
+HIDDEN_COLUMNS = {NATURAL_ORDER_COLUMN_NAME}
 
 
 class _InternalFrame(object):
```

Comment:
I also currently plan to add `_InternalColumn` for Series, Index and MultiIndex, which inherits `_InternalFrame`